Exercise2-Group Hinata.R

```diff
@@ -0,0 +1,24 @@
+# Install and load necessary packages
+install.packages("foreign")
+library(foreign)
+install.packages("gtsummary")
+library(gtsummary)
+library(ggplot2)
```

Comment:
A few things to remember:

* Avoid hard coding the `install.packages()` code into you script as I have previously mentioned. This is not good practice in terms of reproducibility. So, please delete those lines or has them out.
* I don't see any functions in lines of code after this where you use the `foreign` package. So, why call a dependency on this package?
* I don't see any functions in the lines of code after this where you use the `ggplot2` package. So, why call a dependency on this package?